### PR TITLE
update the "grafana-tidb-dashboard.md"

### PR DESCRIPTION
**Pull request by chunleiz**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Some translations have been optimized, the redundant translation of "handshake error OPS" has been deleted, and the translation of "wait duration" has been added

### Which TiDB version(s) do your changes apply to? (Required)

dev

**Tips for choosing the affected version(s):**

CHOOSE MASTER ONLY

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/docs-cn/pull/5753

- This PR is translated from:
https://github.com/pingcap/docs/blob/master/grafana-tidb-dashboard.md

### CODE REVIEW

**Comment by chunleiz** [comment]
translation/from-docs-cn

File: grafana-tidb-dashboard.md [hunk 1]
```diff
@@ -1,144 +1,144 @@
...
-title: TiDB 重要监控指标详解
+title: TiDB 监控指标
+summary: 了解一些Grafana TiDB Dashboard展示的关键指标
 aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-monitoring-metrics/tidb-dashboard/']
 ---
 
 # TiDB 重要监控指标详解
 
-使用 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)，监控架构参见 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
+使用 TiUP 部署 TiDB 集群时，一键部署监控系统 (Prometheus & Grafana)时，参考监控架构 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:9]:
```suggestion
使用 TiUP 部署 TiDB 集群时，你可以一键部署监控系统 (Prometheus & Grafana)，参考监控架构 [TiDB 监控框架概述](/tidb-monitoring-framework.md)。
```

File: grafana-tidb-dashboard.md [hunk 2]
```diff
@@ -1,144 +1,144 @@
...
-    - QPS：所有 TiDB 实例上的每秒执行的 SQL 语句数量。按 `SELECT`、`INSERT`、`UPDATE` 类型进行了区分
-    - CPS By Instance：每个 TiDB 实例上的命令统计。按照命令和执行结果成功或失败来统计
-    - Failed Query OPM：每个 TiDB 实例上，对每秒钟执行 SQL 语句发生的错误按照错误类型的统计（例如语法错误、主键冲突等）。包含了错误所属的模块和错误码
-    - Slow query：慢查询处理时间统计（整个慢查询耗时、Coprocessor 耗时、Coprocessor 调度等待时间），慢查询分为 internal 和 general SQL 语句
-    - Connection Idle Duration：空闲连接的持续时间
-    - 999/99/95/80 Duration：不同类型的 SQL 语句执行耗时统计（不同百分位）
+        - 客户端网络请求发送到 TiDB，到 TiDB 执行结束后返回给客户端的时间。一般情况下，客户端请求都是以 SQL 语句的形式发送，但也可以包含 `COM_PING`、`COM_SLEEP`、`COM_STMT_FETCH`、`COM_SEND_LONG_DATA` 之类的命令执行时间。
+        - 由于 TiDB 支持 Multi-Query，因此，客户端可以一次性发送多条 SQL 语句，如 `select 1; select 1; select 1;`。此时的执行时间是所有 SQL 语句执行完之后的总时间。
+    - Command Per Second：TiDB 按照执行结果成功或失败来统计每秒处理的命令数。
+    - QPS：按 `SELECT`、`INSERT`、`UPDATE` 类型统计所有 TiDB 实例上的每秒执行的 SQL 语句数量。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:25]:
```suggestion
    - QPS：按 `SELECT`、`INSERT`、`UPDATE` 类型统计所有 TiDB 实例上每秒执行的 SQL 语句数量。
```

File: grafana-tidb-dashboard.md [hunk 3]
```diff
@@ -1,144 +1,144 @@
...
+    - CPS By Instance：按照命令和执行结果成功或失败来统计每个 TiDB 实例上的命令。
+    - Failed Query OPM：每个 TiDB 实例上，对每秒钟执行 SQL 语句发生的错误按照错误类型（例如语法错误、主键冲突等）。包含了错误所属的模块和错误码。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:27]:
```suggestion
    - Failed Query OPM：每个 TiDB 实例上，对每秒钟执行 SQL 语句发生的错误按照错误类型进行统计（例如语法错误、主键冲突等）。包含了错误所属的模块和错误码。
```

File: grafana-tidb-dashboard.md [hunk 4]
```diff
@@ -1,144 +1,144 @@
...
+    - Slow query：慢查询处理时间（整个慢查询耗时、Coprocessor 耗时、Coprocessor 调度等待时间），慢查询分为 internal 和 general SQL 语句。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:28]:
```suggestion
    - Slow query：慢查询的处理时间（整个慢查询耗时、Coprocessor 耗时、Coprocessor 调度等待时间），慢查询分为 internal 和 general SQL 语句。
```

File: grafana-tidb-dashboard.md [hunk 5]
```diff
@@ -1,144 +1,144 @@
...
+    - Connection Idle Duration：空闲连接的持续时间。
+    - 999/99/95/80 Duration：不同类型的 SQL 语句执行耗时（不同百分位）。
 
 - Query Detail
-    - Duration 80/95/99/999 By Instance：每个 TiDB 实例执行 SQL 语句的耗时统计（不同百分位）。
-    - Failed Query OPM Detail：每个 TiDB 实例执行 SQL 语句发生的错误按照错误类型统计（例如语法错误、主键冲突等）。
-    - Internal SQL OPS：整个 TiDB 集群内部 SQL 语句执行的 QPS。内部 SQL 语句是 TiDB 内部自动执行的 SQL 语句，一般由用户 SQL 语句来触发或者内部定时任务触发。
+    - Duration 80/95/99/999 By Instance：每个 TiDB 实例执行 SQL 语句的耗时（不同百分位）。
+    - Failed Query OPM Detail：每个 TiDB 实例执行 SQL 语句发生的错误按照错误类型（例如语法错误、主键冲突等）。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:34]:
```suggestion
    - Failed Query OPM Detail：每个 TiDB 实例执行 SQL 语句发生的错误按照错误类型统计（例如语法错误、主键冲突等）。
```

File: grafana-tidb-dashboard.md [hunk 6]
```diff
@@ -1,144 +1,144 @@
...
-    - Queries Using Plan Cache OPS：每秒使用 Plan Cache 的查询数量统计。
+    - Compile Duration：将解析后的 SQL AST 编译成执行计划的耗时。
+    - Execution Duration：执行 SQL 语句执行计划耗时。
+    - Expensive Executor OPS：每秒消耗系统资源比较多的算子。包括 Merge Join、Hash Join、Index Look Up Join、Hash Agg、Stream Agg、Sort、TopN 等。
+    - Queries Using Plan Cache OPS：每秒使用 Plan Cache 的查询数量。
 
 - Distsql
     - Distsql Duration：Distsql 处理的时长
-    - Distsql QPS：Distsql 的数量统计
+    - Distsql QPS：每秒Distsql 的数量
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:87]:
```suggestion
    - Distsql QPS：每秒 Distsql 的数量
```

File: grafana-tidb-dashboard.md [hunk 7]
```diff
@@ -1,144 +1,144 @@
...
     - KV Backoff OPS：TiKV 返回错误信息的数量
     - Lock Resolve OPS：TiDB 清理锁操作的数量。当 TiDB 的读写请求遇到锁时，会尝试进行锁清理
     - Other Errors OPS：其他类型的错误数量，包括清锁和更新 SafePoint
 
 - KV Request
-    - KV Request OPS：KV Request 执行次数，根据 TiKV 显示
-    - KV Request Duration 99 by store：KV Request 执行时间，根据 TiKV 显示
-    - KV Request Duration 99 by type：KV Request 执行时间，根据类型显示
+    - KV Request OPS：KV Request 根据 TiKV 显示执行次数
+    - KV Request Duration 99 by store：根据 TiKV 显示KV Request 执行时间
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:102]:
```suggestion
    - KV Request Duration 99 by store：根据 TiKV 显示 KV Request 执行时间
```

File: grafana-tidb-dashboard.md [hunk 8]
```diff
@@ -1,144 +1,144 @@
...
+    - KV Request Duration 99 by type：根据类型显示KV Request 执行时间
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:103]:
```suggestion
    - KV Request Duration 99 by type：根据类型显示 KV Request 的执行时间
```

File: grafana-tidb-dashboard.md [hunk 9]
```diff
@@ -1,144 +1,144 @@
...
 
 - PD Client
-    - PD Client CMD OPS：PD Client 每秒执行命令数量统计
+    - PD Client CMD OPS：PD Client 每秒执行命令数量
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:106]:
```suggestion
    - PD Client CMD OPS：PD Client 每秒执行命令的数量
```

File: grafana-tidb-dashboard.md [hunk 10]
```diff
@@ -1,144 +1,144 @@
...
     - PD Client CMD Duration：PD Client 执行命令耗时
-    - PD Client CMD Fail OPS：PD Client 每秒执行命令失败统计
+    - PD Client CMD Fail OPS：PD Client 每秒执行命令失败数量
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:108]:
```suggestion
    - PD Client CMD Fail OPS：PD Client 每秒执行命令失败的数量
```

File: grafana-tidb-dashboard.md [hunk 11]
```diff
@@ -152,17 +152,17 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
 
 - GC
     - Worker Action OPM：GC 相关操作的数量，包括 run\_job，resolve\_lock，delete\_range 等操作
-    - Duration 99：GC 相关操作的耗时统计
+    - Duration 99：GC 相关操作的耗时
     - Config：GC 的数据保存时长 (life time) 和 GC 运行间隔 (run interval) 配置
-    - GC Failure OPM：GC 相关操作失败数量统计
+    - GC Failure OPM：GC 相关操作失败数量
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:157]:
```suggestion
    - GC Failure OPM：GC 相关操作失败的数量
```

File: grafana-tidb-dashboard.md [hunk 12]
```diff
@@ -152,17 +152,17 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
...
     - Action Result OPM：GC 相关操作结果数量
     - Delete Range Task Status：Delete range 的任务状态，包含完成和失败状态
     - Push Task Duration 95：将 GC 子任务推送给 GC worker 的耗时
 
 - Batch Client
-    - Pending Request Count by TiKV：等待处理的 Batch 消息数量
-
-    - Batch Client Unavailable Duration 95：Batch 客户端不可用的时间
-    - No Available Connection Counter：Batch 客户端找不到可用链接的次数
+    - Pending Request Count by TiKV： TiKV 批量消息处理等待数量
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:165]:
```suggestion
    - Pending Request Count by TiKV：TiKV 批量消息处理的等待数量
```

File: grafana-tidb-dashboard.md [hunk 13]
```diff
@@ -152,17 +152,17 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
...
+    - Wait Duration 95: 批量消息处理等待时间。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:166]:
```suggestion
    - Wait Duration 95: 批量消息处理的等待时间。
```

File: grafana-tidb-dashboard.md [hunk 14]
```diff
@@ -152,17 +152,17 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
...
+    - Batch Client Unavailable Duration 95： 批处理客户端不可用时长。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:167]:
```suggestion
    - Batch Client Unavailable Duration 95：批处理客户端的不可用时长。
```

File: grafana-tidb-dashboard.md [hunk 15]
```diff
@@ -152,17 +152,17 @@ aliases: ['/docs-cn/dev/grafana-tidb-dashboard/','/docs-cn/dev/reference/key-mon
...
+    - No Available Connection Counter：批处理客户端不可用连接数。
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:168]:
```suggestion
    - No Available Connection Counter：批处理客户端不可用的连接数。
```

**Review by TomShawn — COMMENTED** [review]
some minor comments~

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@chunleiz This PR is not a piece of translation from a PR in the English doc repo. Instead, it is a document enhancement on the Chinese document. Therefore, this PR is to be translated and we should add `translation/doing` label.

File: grafana-tidb-dashboard.md [hunk 16]
```diff
@@ -1,144 +1,144 @@
 ---
-title: TiDB 重要监控指标详解
+title: TiDB 监控指标
+summary: 了解一些Grafana TiDB Dashboard展示的关键指标
```
Comment (TomShawn) [inline grafana-tidb-dashboard.md:3]:
```suggestion
summary: 了解 Grafana Dashboard 中展示的关键指标。
```

**Pull request merged**